misc/remote-backend-on-s3/env0.yml

```diff
@@ -0,0 +1,8 @@
+deploy:
+  steps:
+    terraformInit:
+      before:
+        - aws s3api create-bucket --bucket env0-tf-remote-backend-bucket --region us-east-1
```

Comment:
Maybe only create it if it doesn't exist?

Reply:
`create-bucket` command is fault tolerant by default (i.e if bucket exists it doesn't fail)